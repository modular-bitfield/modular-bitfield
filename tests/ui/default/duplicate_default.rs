use modular_bitfield::prelude::*;

#[bitfield]
pub struct DuplicateDefault {
    #[default = true]
    #[default = false]
    pub field: bool,
}

fn main() {}