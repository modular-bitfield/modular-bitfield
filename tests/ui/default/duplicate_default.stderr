error: encountered duplicate `#[default = ...]` attribute for field
 --> tests/ui/default/duplicate_default.rs:6:7
  |
6 |     #[default = false]
  |       ^^^^^^^^^^^^^^^

error: duplicate `#[default = ...]` here
 --> tests/ui/default/duplicate_default.rs:5:7
  |
5 |     #[default = true]
  |       ^^^^^^^^^^^^^^
